File: java/fizzbuzz.java
public class fizzbuzz {
	public static void main(String[] args){
		for( int i = 0; i < 100; i++ ){
			String msg = "";
			if( i % 3 == 0 ){
				msg += "fizz";
			}
			if( i % 5 == 0 ){
				msg += "buzz";
			}
			if(msg.equals("")){
				msg = "" + i;
			}
			System.out.println(msg);
		}
	}
}

File: php/fizzbuzz.php
<?php
	for( $i = 0; $i < 100; $i++){
		$msg = "";
		if($i % 3 == 0){
			$msg = $msg . "fizz";
		}
		if($i % 5 == 0){
			$msg = $msg . "buzz";
		}
		if($msg == ""){
			$msg = $i;
		}
		echo "$msg\n";
	}
?>